docs/pages/modules/supporting-platforms.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Supporting more platforms
+description: Learn how to add support for macOS and tvOS platforms.
+---
+
+import { Step } from '~/ui/components/Step';
+
+Expo Modules API provides the first-class support for Android and iOS, however since all Apple platforms are based on the same foundation and use the same programming language,
+it is possible to target other Apple platforms in the Expo module. At the moment only **macOS** and **tvOS** platforms (also called [Out-of-Tree platforms](https://reactnative.dev/docs/out-of-tree-platforms)) are supported.
+This guide will walk you through adding support for these platforms.
```

Comment:
```suggestion
Expo Modules API provides first-class support for Android and iOS. 

At the moment, only **macOS** and **tvOS** platforms (also called [Out-of-Tree platforms](https://reactnative.dev/docs/out-of-tree-platforms)) are supported. However, since all Apple platforms are based on the same foundation and use the same programming language, targeting other Apple platforms in the Expo module is possible. This guide will walk you through adding support for these platforms.
```

- Let Prettier handle the formatting. 
- Also, let's split paragraph where it is described about what platforms are supported

Reply:
I think your suggestion has a little bit different meaning and is confusing for me. What about this?

```suggestion
Expo Modules API provides first-class support for Android and iOS. However, since all Apple platforms are based on the same foundation and use the same programming language, targeting other [Out-of-Tree platforms](https://reactnative.dev/docs/out-of-tree-platforms) in the Expo module is possible. At the moment, only **macOS** and **tvOS** platforms are supported. This guide will walk you through adding support for these platforms.
```

Reply:
I see your point. Yes, this sounds good to me 👍